mailpoet/assets/js/src/automation/integrations/mailpoet/components/form-token-field/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { FormTokenField as WpFormTokenField } from '@wordpress/components';
+import { uniq } from 'lodash';
```

Comment:
This is not important now, but could we use just a native `Set`, for instance?
I think there's a tendency to get rid of loadash/underscore in the JS ecosystem as modern JS offers most of that functionality quite out-of-the-box. In Gutenberg repo, I've seen such an initiative.

Reply:
A more troubling question is, I'm realizing, why do we need this, and what happens with two lists of the same name then?

Reply:
Problem outdated thanks to [710190b](https://github.com/mailpoet/mailpoet/pull/4371/commits/710190bd68357719381a9d577bc886a495f5a367)

Reply:
710190b is a bit of a bigger change. Instead of having an "Any item" option, I changed the behavior to make "Any item" the placeholder. This solved my mentioned "Any item has a `x` to close it, but because it's default it does not get deleted"-problem. It also simplified much of the used logic and some of the problems, you are mentioning further down.